src/operator/effectivepolicy/groupreconciler.go

```diff
@@ -77,6 +77,9 @@ func (g *GroupReconciler) getAllServiceEffectivePolicies(ctx context.Context) ([
 		services.Add(service)
 		serviceToIntent[service] = clientIntent
 		for _, intentCall := range clientIntent.GetCallsList() {
+			if intentCall.Type == v1alpha3.IntentTypeInternet {
+				continue
```

Comment:
This took me a while to understand, can you add a comment here? It seems to be because we don't want to consider the target, the Internet, as a service.